boards/common/stm32/dist/stm32l0.cfg

```
@@ -1,3 +1,8 @@
-source [find target/stm32l0.cfg]
+# Try to use dual bank if available and supported
+try {
+    source [find target/stm32l0_dual_bank.cfg]
+} on error {} {
+    source [find target/stm32l0.cfg]
```

This detects it and tell the user

```suggestion
    error "Openocd version does not support 'stm32l0_dual_bank' please upgrade to a version supporting it"
```

---

Would it be better to use puts instead of error, that way someone with old opeocd (someone that doesn't build it form source for example) can still flash the board.

`puts "Openocd version does not support 'stm32l0_dual_bank' please upgrade to a version supporting it, falling back to single bank stm32l0.cfg"`

---

Suggestion for a bit of editing:
`Your Openocd version does not support the dual bank flash on your board. Falling back to single bank.`
?

---

Would then add a "WARNING: " at the beginning at least.